### ISSUE TITLE
Allow for adding/overriding loaders

### ISSUE THREAD

**Issue by whitlockjc** [whitlockjc/path-loader#1]
It would be slick to allow for the consumer to add/replace loaders.

**Comment by kristianmandrup** [comment]
Yes, exactly. I'd like to use this to add support for loading relative to a base path passed in options.
